scripts/ci-frontend-metrics.sh

```diff
@@ -1,25 +1,30 @@
 #!/bin/bash
 set -e
 
-ERROR_COUNT="$(./node_modules/.bin/tsc --project tsconfig.json --noEmit --strict true | grep -oP 'Found \K(\d+)')"
+ERROR_COUNT="$(yarn run tsc --project tsconfig.json --noEmit --strict true | grep -oP 'Found \K(\d+)')"
```

Comment:
`yarn run` will run anything from the .bin directory, so we don't need to call it like this manually ✨